src/test.js

```diff
@@ -728,7 +728,7 @@ export function internalStop( test ) {
 }
 
 // Forcefully release all processing holds.
-function internalRecover( test ) {
+export function internalRecover( test ) {
```

Comment:
I'm not sure I understand what you mean here.

Would it be worth creating an issue to explain this in more detail (as well as to help track it so we eventually fix it)?